### PR TITLE
Use environment ids for cca resources

### PR DESCRIPTION
**Pull request by franzpgarcia**

### CODE REVIEW

File: cloudca/README.md [hunk 1]
```diff
@@ -99,17 +95,15 @@ Create a network ACL.
 ### Example usage
 ```hcl
 resource "cloudca_network_acl" "my_acl" {
-	service_code = "compute-qc"
-	environment_name = "dev"
+   environment_id = "4cad744d-bf1f-423d-887b-bbb34f4d1b5b"
 	name = "test-acl"
```
Comment (franzpgarcia) [inline cloudca/README.md]: Will fix formatting once review is done

File: cloudca/resource_cloudca.go [hunk 2]
```diff
@@ -65,3 +66,17 @@ func getResources(d *schema.ResourceData, meta interface{}) cloudca.Resources {
...
 }
+
+// Deals with all of the casting done to get a cloudca.Resources.
+func getResourcesForEnvironmentId(d *schema.ResourceData, meta interface{}) (cloudca.Resources, error) {
+	client := meta.(*cca.CcaClient)
+	environment, err := client.Environments.Get(d.Get("environment_id").(string))
+	if err != nil {
+		return cloudca.Resources{}, err
+	}
+	resources, err2 := client.GetResources(environment.ServiceConnection.ServiceCode, environment.Name)
```
Comment (prollynomial) [inline cloudca/resource_cloudca.go]: If I remember correctly, `err2` can just be `err`. No need to introduce a new variable.

Comment (franzpgarcia) [inline cloudca/resource_cloudca.go]: Fixed

File: cloudca/README.md (hunk 1 above)
Comment (pdube) [inline cloudca/README.md]: Why wait?

File: cloudca/resource_cloudca.go [hunk 3]
```diff
@@ -65,3 +66,17 @@ func getResources(d *schema.ResourceData, meta interface{}) cloudca.Resources {
 	_resources, _ := client.GetResources(d.Get("service_code").(string), d.Get("environment_name").(string))
 	return _resources.(cloudca.Resources)
 }
+
+// Deals with all of the casting done to get a cloudca.Resources.
+func getResourcesForEnvironmentId(d *schema.ResourceData, meta interface{}) (cloudca.Resources, error) {
```
Comment (pdube) [inline cloudca/resource_cloudca.go]: Why pass in the full ResourceData?

File: cloudca/README.md (hunk 1 above)
Comment (franzpgarcia) [inline cloudca/README.md]: Because I am replacing the tabs by spaces. That will modify all the fields.

File: cloudca/resource_cloudca.go (hunk 3 above)
Comment (franzpgarcia) [inline cloudca/resource_cloudca.go]: Fixed

File: cloudca/README.md (hunk 1 above)
Comment (pdube) [inline cloudca/README.md]: Got it

**Pull request merged**